src/AppBundle/Resources/views/App/userProfile.html.twig

```diff
@@ -0,0 +1,96 @@
+{% extends "AppBundle::layout.html.twig" %}
+
+{% block app_body %}
+    <!DOCTYPE html>
+    <html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        {% block stylesheets %}
+            {{ parent() }}
+            <link href="{{ asset('assets/css/style.css') }}" rel="stylesheet" />
+        {% endblock %}
+        <title>{% block title %}Profile Admin{% endblock %}</title>
```

Comment:
why do you need to include all headers to update the title and/or css check https://github.com/Studeal/sqldashboard/pull/2/files